scripts/cli/src/util/init.ts

```diff
@@ -79,10 +79,12 @@ export async function disconnect() {
 export async function sleep(ms: number) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
-const getEra = async () =>
-  (await (await ApiSingleton.getInstance()).query.staking.activeEra())
-    .unwrap()
-    .index.toJSON();
+const getEra = async () => {
+  const api = await ApiSingleton.getInstance();
+  const activeEra = <Option<ActiveEraInfo>>(await api.query.staking.activeEra());
```

Comment:
Shouldn't need the type cast, since the type should already be `Option<ActiveEraInfo>`
```suggestion
  const activeEra = await api.query.staking.activeEra();
```